README.md

```diff
@@ -128,6 +122,19 @@ cd rap-package-template
 python examples/example_create_publication.py
 ```
 
+## Using GitHub codespaces
+
+If you are using GitHub Codespaces, the above installation steps will be completed automatically, so you don't need to do anything! 
+
+Click the "Code" button above, click the "Codespaces" tab, and then click the "+" button to create a new codespace. The environment may take a minute or two to build when you load it for the first time.
+
+## Running the pipeline
+Before running the pipeline, make sure you are in the same folder as the `create-publication.py` file by entering the following command into the terminal:
+`cd RAP_example_pipeline_python`
+
+To run the pipeline, enter the following command into the terminal:
+`spark-submit --master local create_publication.py`
```

Comment:
interesting - I was able to run it by just typing `python create_publication.py`

Reply:
oh! didn't know that, I'll update as that's simpler